Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/migration/impl/ModelMapperUtil.java

```diff
@@ -0,0 +1,131 @@
+package edu.asu.conceptpower.app.migration.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+import edu.asu.conceptpower.core.ConceptEntry;
+import edu.asu.conceptpower.core.ConceptList;
+import edu.asu.conceptpower.core.ConceptType;
+import edu.asu.conceptpower.core.ReviewRequest;
+import edu.asu.conceptpower.servlet.core.ChangeEvent;
+
+/**
+ * 
+ * @author Keerthivasan
+ * 
+ */
+
+public class ModelMapperUtil {
+    
+    /*Avoid Instantiation of this utility class*/
+    private ModelMapperUtil() {}
+    
+    public static List<edu.asu.conceptpower.app.core.model.impl.ConceptType> conceptTypeMapper(ConceptType[] conceptTypeList){  
```

Comment:
we're using spring, so no static methods. make this class a component or service.

Reply:
also, I would not pass an array. you would end up holding two potentially very long arrays/lists in memory instead of doing it one by one. (meaning you would probably still have one big array, but you could write them one by one into the database to avoid having twice the amount of objects in memory).

Reply:
and last, `conceptTypeMapper` is a bad method name. Method names are usually verbs.

Reply:
this is try for all the other methods as well.